src/lib/components/CarouselWidget.svelte

```diff
@@ -9,31 +9,30 @@
 	let className = '';
 	export { className as class };
 	let currentIndex = 0;
-	// Function to move to the next slide
 	function nextSlide() {
 		currentIndex = (currentIndex + 1) % pictures.length;
 	}
+
 	let interval: number;
```

Comment:
```suggestion
	let interval: ReturnType<setInterval>;
```

And remove the `Number()` casts, it's not guaranteed to work